Calibre import. Fixes #627

Ryuno-Ki opened the pull request.

Ryuno-Ki commented:
What is needed to push this PR forward?
Or are you simply swamped and need some time?

mouse-reeve commented:
I just need some time -- I'm dealing with some outside life stuff right now that has temporarily tanked my ability to do more than respond to questions at the moment. But I'm excited to review this when I have a brain that can read code back in my head :)

Ryuno-Ki commented:
Alright. Take your time.

I'm looking into picking up another issue in the meantime.

Ryuno-Ki commented on bookwyrm/importers/calibre_import.py:
```diff
@@ -0,0 +1,12 @@
+
+    def get_shelf(self, normalized_row):
+        # Calibre export does not indicate which shelf to use. Go with a default one for now
+        return "to-read"
```
Could be `models.Shelf.TO_READ` here (and similar in other importers).

mouse-reeve replied:
that would be better!

Ryuno-Ki replied:
Updated. Added a test (and updated the importer tests) while I was at it.

mouse-reeve reviewed (changes requested):
(The requested changes are the comment above, I'm just updating the state so it's easier for me to keep track of PRs)

Ryuno-Ki commented:
I believe I've addressed the issue.
When importing, I got another error about the imported date already containing a timezone (it is aware, according to pytz) and addressed it, too.

However, I couldn't run black here due to https://stackoverflow.com/questions/71673404/importerror-cannot-import-name-unicodefun-from-click

mouse-reeve commented:
Looks great! Thanks so much for the fixes
